src/components/Field/index.js

```diff
@@ -41,144 +45,139 @@ class Field extends Component {
 
     isComplete = () => {
         let state = this.state.cards.every(el => {
```

Comment:
`const areCardsRemoved = this.state.cards.every(card => card.removed)` or something like that